README.md

```diff
@@ -1,4 +1,5 @@
 # Poglink
+[![codecov](https://codecov.io/gh/FM-17/poglink/branch/main/graph/badge.svg?token=Y84IU5NAH0)](https://codecov.io/gh/FM-17/poglink)
```

Comment:
I think I'm going to start adding more badges here because I think they look pro